src/index.ts

```diff
@@ -22,7 +22,7 @@ interface GraphData {
   currentNoteID: string;
   nodeFontSize: number;
   nodeDistanceRatio: number;
-  isIncludeBacklinks: boolean;
+  isShowLinkDirectionArrows: boolean;
```

Comment:
do you mind renaming this `shouldShowLinkDirection`?

Reply:
Good point you catch here. Consistent naming convention is ... hard :-). 
`shouldXXX` sounds like you express a preference, and the system will decide ultimately. Instead my suggestion would be either 
a) to always prepend `is` for boolean attributes - `isIncludeBacklinks`, `isShowLinkDirectionArrows` or
b) simply use more fluent language - `includeBacklinks`, `showLinkDirectionArrows`.

I am in favor of b). What do you think?

Reply:
Good points, let's go with b.  I liked the prefix so it was easy to know it's a bool, but now we have TypeScript annotations for that :)